website/assets/apps/megamap/helpers/Cartographer.js

```diff
@@ -306,6 +306,13 @@ export default class Cartographer {
     // Header row
     const header = document.createElement('div');
     header.setAttribute('class', 'popup-header');
+    
+    // A container element to hold elements that *can* trigger
```

Comment:
I think this is where you are add the clickable behavior to the text as well as the zoom icon.  It works, but I still get a "text edit" style mouse pointer when I hover over the text.  Should that indicate, "Hey, this is clickable"